src/cloudlabs/templates/add_host.html

```diff
@@ -32,6 +32,7 @@
     </fieldset>
     <fieldset>
       <legend>Machine setup</legend>
+      {{ render_field(form.setup_script) }}
```

Comment:
Maybe put this one after the port? So the user enters the repo, the port and the setup script.